build/publish.sh

```diff
@@ -33,3 +33,4 @@ git checkout master
 git branch -D build
 
 echo "All done."
+echo "Remember to run 'npm run-script integrity' to upload the information in the website's download page"
```

Comment:
The `integrity` task doesn't actually upload the information, or am I missing something? So the text should probably read something like

```
Remember to run 'npm run-script integrity' _and_ upload the information in the website's download page
```

(Added emphasis to show the difference.)